flagsmith/flagsmith.py

```diff
@@ -5,9 +5,13 @@
 import requests
 from flag_engine import engine
 from flag_engine.environments.models import EnvironmentModel
-from flag_engine.identities.models import IdentityModel, TraitModel
+from flag_engine.identities.models import (  # type: ignore
+    IdentityModel,
+    TraitModel,
+)
```

Comment:
Just out of curiosity why was `type: ignore` necessary here?

Reply:
Update the `type: ignore` to be more specific.

Reply:
@tushar5526 Could you change the import to `flag_engine.identities.traits.TraitModel` instead?

Reply:
Done, mypy is not throwing any errors. Marking this as resolved.